imports/ui/unit-wizard/unit-wizard.jsx

```diff
@@ -151,27 +173,16 @@ class UnitWizard extends Component {
                 disabled={inProgress}
                 onChange={this.createTextStateHandler('city')}
               />
-              <SelectField
+              <AutoComplete
```

Comment:
You should make use of the "searchText" attribute to make this input a controlled input (all of our inputs are controlled inputs). Please see the "Controlled Example" on https://v0.material-ui.com/#/components/auto-complete

Reply:
Thank you! onNewRequest={this.handleNewRequest} is used to update this input as controlled input. "searchText" value is reset with the `onNewRequest` callback.

Reply:
Hey @sophieyang0813 , a controlled input is one that both sets and gets its value from a state variable (two-way binding). Currently it only sets (one-way binding). Please refer to the example I mentioned

Reply:
https://reactjs.org/docs/uncontrolled-components.html
Please consider my previous request again and use the "searchText" attribute